src/ext/java/org/opentripplanner/ext/siri/AddedTripBuilder.java

```diff
@@ -236,11 +244,26 @@ Result<TripUpdate, UpdateError> build() {
       return TripTimesValidationMapper.toResult(tripId, validityResult.get());
     }
 
+    var tripOnServiceDate = TripOnServiceDate
+      .of(tripId)
```

Comment:
Question: This will create a `TripOnServiceDate` with the same id as the `Trip` we add. Is this ok?

Reply:
An ID for a `Trip` added from SIRI ET (a.k.a. ExtraJourney in SIRI terms) is unique and only exists for a single day. In effect this implies that the ID can also be interpreted as a `TripOnServiceDate`. I do not have enough insight in the OTP datamodel to vouch for any other side-effects of having a `Trip` and a `TripOnServiceDate` with the same ID, but conceptually, I see no problem with it.

The code that was removed in #5140 always replaced the `TripOnServiceDate` in the plan-data/graph with whatever was sent in the SIRI-data - regardless if it was an added trip or a normal update. When the SIRI-data was from an external source with a separate set of ids, we ended up with a having lots of completely unknown `TripOnServiceDate`-references. As long as this _only_ applies to added journeys that do not already exist, this should not be an issue.

Reply:
@lassetyr Thanks, then it seems all fine. This only adds TripOnServiceDate for added trips, not for regular ET messages.